src/main/java/hydroguard/api/mock/controllers/sensors/SensorController.java

```diff
@@ -0,0 +1,47 @@
+package hydroguard.api.mock.controllers.sensors;
+
+import hydroguard.api.mock.models.sensors.SensorDTO;
+import org.springframework.web.bind.annotation.*;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/sensors")
+public class SensorController {
+    private final List<SensorDTO> sensors = new ArrayList<>();
+
+    @GetMapping("/{id}")
+    public SensorDTO getSensorById(@PathVariable UUID id) {
+        return sensors.stream().filter(sensor -> sensor.getId().equals(id)).findFirst().orElse(null);
+    }
+
+    @PostMapping
+    public SensorDTO createSensor(@RequestBody SensorDTO sensor) {
```

Comment:
Hoe er ook rekening mee dat je in deze fgunction nooit de crezted at en updated at set

Reply:
Wat bedoel je precies? Dat ze hier aan gemaakt moeten worden? Of juist wat anders?

Reply:
Ik bedoelde dat wanneer het object wordt aangemaakt hier, wordt de created at en updated at aangepast naar de tijd van nu of niet? Of zijn deze nog op hun default value. Kan het niet nu ff controleren want ben op mobile. Maar ik dacht dat als ik een nieuw object toevoegde dat die 2 values op hun default lowest value stonden.

Reply:
Als je de post doet dan wordt de created/updated at op de tijd gezet dat je hem aanmaakt

Reply:
Top, vanuit de code was dat een beetje onduidelijk, maar als dat klopt is er niets mis!